app/controllers/api/v1x2/mixins/index_mixin.rb

```diff
@@ -3,6 +3,11 @@ module V1x2
     module Mixins
       module IndexMixin
         include Api::V1x1::Mixins::IndexMixin
+
+        def permitted_params
```

Comment:
It's called when return a collection of order process, to allow 3 extra parameters. The reason of approval-api doesn't have this change is because it overwrites `filter` method to avoid such parameters checking.